lib/data_update_scripts/20200723070918_update_articles_cached_organization.rb

```diff
@@ -0,0 +1,15 @@
+module DataUpdateScripts
+  class UpdateArticlesCachedOrganization
+    ATTRIBUTES = %i[name slug username profile_image_90 profile_image_url].freeze
+
+    def run
+      Article.find_each do |article|
```

Comment:
To make this run faster we should filter for only articles that have organizations.

Reply:
We don't seem to index articles on `organization id`, only `user_id` so this would still be a seq scan (from Blazer):

```sql
Seq Scan on articles  (cost=0.00..207773.65 rows=23879 width=2252) (actual time=0.013..307.515 rows=17537 loops=1)
  Filter: (organization_id IS NOT NULL)
  Rows Removed by Filter: 313960
Planning Time: 0.191 ms
Execution Time: 308.365 ms
```

It would however save us from instantiating unnecessary objects and that's definitely a plus, so thanks! The new version of this script now filters by presence of  `cached_user` or `cached_organization`. Are there any statement timeouts on the live DB I need to worry about? I do hit them on Blazer, but I think that's a Blazer limit.